### PR TITLE
Adding UMBC faculty: 8 new faculty, 1 affiliation change, 1 alias addition for existing entry

### PR DESCRIPTION
**Pull request by tejas-gokhale**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (up to three months, as processing has moved to a quarterly cadence).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR COMMIT MAY BE SUMMARILY REJECTED.

1. _Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv")._

1. _Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv`._

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

1. _Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files._ **Do not modify `csrankings.csv`, which is auto-generated.**

1. _Check to make sure that you have no spaces after commas, or any missing fields._

1. _Check to make sure the home page is correct._

1. _Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`)._

1. _Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org)._

1. _If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below)._

1. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [x] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ database. Use the GitHub user interface or a text editor like emacs or NotePad i
 
 **Inclusion criteria**
 
-- [ ] Make sure that any faculty you add meet the inclusion
+- [X] Make sure that any faculty you add meet the inclusion
```
Comment (emeryberger) [inline CONTRIBUTING.md]: This file should not be part of this PR, and needs to be removed (not "deleted", which would delete it from the repository).

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (tejas-gokhale) [reply CONTRIBUTING.md]: Hi Emery, I'm not quite sure how to do that -- could you help me here?

Comment (emeryberger) [reply CONTRIBUTING.md]: I suggest you close this PR and open a new PR without `CONTRIBUTING.md`.

Comment (tejas-gokhale) [reply CONTRIBUTING.md]: I was able to remove the file from the PR.  Let me know if this works.  Thanks!

File: csrankings-k.csv [hunk 2]
```diff
@@ -759,6 +760,7 @@ Kostas Alexis,NTNU,http://www.kostasalexis.com,1rPSVNAAAAAJ
 Kostas Blekas,University of Ioannina,https://cs.uoi.gr/~kblekas,dgH4XJkAAAAJ
 Kostas Daniilidis,University of Pennsylvania,http://www.cis.upenn.edu/~kostas,dGs2BcIAAAAJ
 Kostas E. Bekris,Rutgers University,http://www.pracsyslab.org/bekris,gwC7rCUAAAAJ
+Kostas Kalpakis,Univ. of Maryland,https://redirect.cs.umbc.edu/~kalpakis/,ktn2QM4AAAAJ
```
Comment (emeryberger) [inline csrankings-k.csv]: Not an entry in DBLP.

File: csrankings-l.csv [hunk 3]
```diff
@@ -51,6 +51,8 @@ Lantao Liu,Indiana University,http://homes.sice.indiana.edu/lantao,L5dHk5cAAAAJ
 Lap Chi Lau,University of Waterloo,https://cs.uwaterloo.ca/~lapchi,umOOMzsAAAAJ
 Lap-Fai Craig Yu,George Mason University,http://www.cs.umb.edu/~craigyu,p_Un9ssAAAAJ
 Lap-Fai Yu,George Mason University,http://www.cs.umb.edu/~craigyu,p_Un9ssAAAAJ
+Lara Martin,Univ. of Maryland - Baltimore County,https://laramartin.net/,YjiWURYAAAAJ
```
Comment (emeryberger) [inline csrankings-l.csv]: Not a DBLP entry.

File: csrankings-l.csv [hunk 4]
```diff
@@ -392,6 +394,7 @@ Li Weigang 0001,Universidade de Brasília,http://cic.unb.br/~weigang,Sw_i7yQAAAA
 Li Xiao 0001,Michigan State University,http://www.cse.msu.edu/~lxiao,NOSCHOLARPAGE
 Li Xiao 0004,University of Aberdeen,https://www.abdn.ac.uk/people/xiao.li,dZLlkYQAAAAJ
 Li Xiong 0001,Emory University,https://cs.emory.edu/~lxiong,jJ8BLgsAAAAJ
+Li Yan,Univ. of Maryland - Baltimore County,https://www.csee.umbc.edu/people/faculty/li-yan/,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings-l.csv]: This is a disambiguation DBLP entry. Either it needs to be augmented with the appropriate suffix or you need to follow the directions on communicating with dblp@dblp.org.

File: csrankings-r.csv [hunk 5]
```diff
@@ -443,6 +444,7 @@ Rebecca J. Passonneau,Pennsylvania State University,https://www.eecs.psu.edu/dep
 Rebecca M. Willett,University of Chicago,https://voices.uchicago.edu/willett,bGRVPl8AAAAJ
 Rebecca N. Wright,Columbia University,https://www.cs.columbia.edu/~rwright,L-xeJPoAAAAJ
 Rebecca Willett,University of Chicago,https://voices.uchicago.edu/willett,bGRVPl8AAAAJ
+Rebecca Williams,Univ. of Maryland - Baltimore County,https://www.csee.umbc.edu/people/faculty/rebecca-williams/,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings-r.csv]: This is not the correct Rebecca Williams entry in DBLP.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-k.csv (hunk 2 above)
Comment (tejas-gokhale) [reply csrankings-k.csv]: removed

File: csrankings-l.csv (hunk 3 above)
Comment (tejas-gokhale) [reply csrankings-l.csv]: removed

File: csrankings-l.csv (hunk 4 above)
Comment (tejas-gokhale) [reply csrankings-l.csv]: removed

File: csrankings-r.csv (hunk 5 above)
Comment (tejas-gokhale) [reply csrankings-r.csv]: corrected to Rebecca M. Williams
